parser/src/config/csv.rs

```diff
@@ -1,5 +1,5 @@
 //! Configuration related to the various character-separated formats, like CSV.
-use super::EncodingRef;
+use super::{EncodingRef, ErrorThreshold};
```

Comment:
I think it makes sense to keep it at the top level, since it might also get used for some other framed formats. I don't anticipate there will be many of those, but I do think there's probably going to be one or two more that do.